exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter.h

```diff
@@ -36,11 +43,22 @@ class OtlpGrpcExporter final : public opentelemetry::sdk::trace::SpanExporter
    */
   OtlpGrpcExporter();
 
+  /**
+   * Create an OtlpGrpcExporter using specified OtlpGrpcClient.
+   *
+   * @param options options to create exporter
+   * @param client the gRPC client to use
+   */
+  OtlpGrpcExporter(const OtlpGrpcExporterOptions &options,
+                   nostd::shared_ptr<OtlpGrpcClient> client);
```

Comment:
nit - use `std::shared_ptr` across the OTLP code.

Reply:
Just want to keep the same type with the parameter in factory.